src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -109,13 +112,9 @@ private bool ActiveDocumentFilter(object viewModel)
             return PathHelper.IsMatchingPath(issueFilePath, activeDocumentFilePath);
         }
 
-        private void SetDefaultSortOrder()
-        {
-            var collectionView = CollectionViewSource.GetDefaultView(Issues);
-
-            collectionView.SortDescriptions.Add(
+        private void SetDefaultSortOrder() =>
```

Comment:
Any chance that the SortDescriptions xaml binding will work now?
https://stackoverflow.com/a/1280768/339720

Reply:
Perhaps, but that example is actually creating the collection in XAML so I don't think that particular example would work.